docs/_static/nuts-auth.yaml

```diff
@@ -116,6 +116,70 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Contract"
+  /auth/accesstoken:
```

Comment:
Would this also be the API to just validate the bearer token and have a vendor generate its own accesstokens?

Reply:
If a vendor wants a different form of authentication than the oauth2 flow, it should implement its own authorization server. The access token is opaque so it can be anything. Currently there is no seperate endpoint to only check the bearer token. Before making such endpoint, I would like a clear use-case.